pkg/auth/dependency/mfa/template.go

```diff
@@ -24,4 +24,11 @@ var TemplateMFAOOBCodeEmailTXT = template.Spec{
 var TemplateMFAOOBCodeEmailHTML = template.Spec{
 	Type:   TemplateItemTypeMFAOOBCodeEmailHTML,
 	IsHTML: true,
+	Default: `<!DOCTYPE html>
+<html>
+<body>
+<p>Your MFA code is: {{ .code }}</p>
```

Comment:
Your {{ .appname }} Two Factor Auth Verification code is: {{ .code }}

Please enter the Verification Code on the Sign-in screen. 

Please ignore this code if this Sign-In was not initated by you.

Reply:
https://github.com/SkygearIO/skygear-server/issues/1161